src/app/main/service/habit-assign/habit-assign.service.spec.ts

```diff
@@ -0,0 +1,125 @@
+import { LanguageService } from './../../i18n/language.service';
+import { TestBed } from '@angular/core/testing';
+import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
+import { HabitAssignService } from './habit-assign.service';
+import { habitAssignLink } from '../../links';
+import {
+  listOfHabits,
+  newHabit,
+  assignResponce,
+  modifiedAssignResponce,
+  HabitsForDate,
+  habitsWithTheSameHabitId
+} from '../../mocks/habit-assign-mock';
+
+describe('HabitService', () => {
+  let service: HabitAssignService;
+  let httpMock: HttpTestingController;
+  const languageServiceMock = jasmine.createSpyObj('LanguageService', ['getCurrentLanguage']);
+  languageServiceMock.getCurrentLanguage = () => 'en';
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [HabitAssignService, { provide: LanguageService, useValue: languageServiceMock }],
+      imports: [HttpClientTestingModule]
+    }),
+      (httpMock = TestBed.get(HttpTestingController));
```

Comment:
Looks like a mistake
After the `configureTestingModule' should be a semicolon
And why httpMock is in brackets?

Reply:
This changing made prettier